plugin/storage/es/spanstore/reader.go

```diff
@@ -159,14 +165,17 @@ func getTimeRangeIndexFn(archive, useReadWriteAliases bool) timeRangeIndexFn {
 }
 
 func getSourceFn(archive bool, maxNumSpans int) sourceFn {
-	return func(query elastic.Query, nextTime uint64) *elastic.SearchSource {
+	return func(query elastic.Query, fields searchAfterFields) *elastic.SearchSource {
 		s := elastic.NewSearchSource().
 			Query(query).
 			Size(defaultDocCount).
 			TerminateAfter(maxNumSpans)
 		if !archive {
-			s.Sort("startTime", true).
-				SearchAfter(nextTime)
+			s.Sort(startTimeField, true).
```

Comment:
shouldn't this be `s = s.Sort()...`, to follow the builder pattern above?

Reply:
No we just add sort if the search is not running for the archive index.